templates/ntp.conf.j2

```diff
@@ -0,0 +1,22 @@
+# {{ ansible_managed }}
```

Comment:
Note that there are some other parameters in both the default configs for Ubuntu and CentOS that I don't want to leave out, since they're there for good reason; I'm going to do some diffs of all four supported OS releases and make sure none of the configs get broken or less secure if the user of this role chooses a managed file.

Unfortunately we can't just use `lineinfile` on this file (at least not in any simple way... maybe there is a way), so a template is probably the simplest solution.